.github/workflows/tests.yml

```
@@ -6,7 +6,13 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - macos-12
```

Should probably use latest for consistency:
```suggestion
          - macos-latest
```

---

Good point, I copied it from one of the other workflows.

---

When I think about it: maybe pin and be a bit more precise to the users on what OS versions this lib supports?

---

Tests for Macos fail due to some lib missing. Did a bit of reading on how to solve it and maybe using `latest` is a solution here. Will give it a try.

---

That seems to have done the trick 🎉